_dev/src/components/product-feed-page/product-feed-pre-scan-table-status-details.vue

```diff
@@ -42,112 +42,130 @@
           id="filterByCountryDropdown"
           variant=" "
           menu-class="ps-dropdown"
-          :text="langSelected ? langSelected
-            : $t('productFeedSettings.shipping.filterTitle')"
-          class="mb-2 mt-1 mt-md-0 ps-dropdown psxmarketingwithgoogle-dropdown bordered maxw-sm-250"
+          :text="langChosen ? langChosen
+            : $t('productFeedSettings.preScan.selectLanguage')"
+          class="mt-1 mb-2 mt-md-0 ps-dropdown psxmarketingwithgoogle-dropdown bordered maxw-sm-250"
         >
           <b-dropdown-item
-            :disabled="!langSelected"
+            :disabled="!langChosen"
             variant="dark"
             link-class="flex-wrap px-3 d-flex flex-md-nowrap align-items-center"
-            @click="langSelected = null"
+            @click="langChosen = null"
           >
             {{ $t('productFeedSettings.shipping.filterTitle') }}
           </b-dropdown-item>
           <b-dropdown-item
-            :disabled="country === langSelected"
-            v-for="(country, index) in countries"
+            :disabled="lang === langChosen"
+            v-for="(lang, index) in countries"
             :key="index"
-            @click="langSelected = country"
+            @click="langChosen = lang"
             variant="dark"
             link-class="flex-wrap px-3 d-flex flex-md-nowrap align-items-center"
           >
-            {{ country }}
+            {{ lang }}
           </b-dropdown-item>
         </b-dropdown>
       </div>
       <b-table
         :items="items"
         :fields="fields"
         :filter-function="filterByLang"
-        :filter="langSelected"
-        :per-page="perPage"
+        :filter="langChosen"
+        :per-page="limit"
         :current-page="currentPage"
         :busy="loading"
         id="table-products"
-        class="ps_gs-table-products mb-3"
+        class="mb-3 ps_gs-table-products"
         table-class="border-bottom-0"
         variant="light"
         responsive="xl"
       >
         <template #cell(id)="data">
-          {{ data.value }}
+          {{ data.item.productId }}
         </template>
         <template #cell(name)="data">
           <a
             class="external_link-no_icon"
+            :class="{'m-3' : data.item.attributeId > 0}"
             :href="!isNaN(data.item.id)
```

Comment:
I'm no sure what you're trying to verify here, but maybe you 'd better check if data.item.id exists and `Number(data.item.id)` to be sure it's a number ?